autolab/docker
# Tango config

How to test:
- checkout Tango submodule's `manage-config` branch. 
- run `docker-compose build  &&  docker-compose up`. Able to see autolab running on localhost.
